### DISCUSSION TITLE
Is it possible to script openOutpaint operations?

### DISCUSSION THREAD

**Discussion by mikoro** [zero01101/openOutpaint#207] [General]
Say I have always the same things I need to do: split an image with the selection tool, move and rotate things around, and then dream a specific area. Can this be scripted?

**Comment by zero01101** [comment]
so there's nothing currently for doing anything of the sort, but thanks to the forward-thinking applied by @seijihariki to make the history system work with the majority of actions that can be performed, i kind of think it's _probably_ plausible?  i mean don't hold your breath or anything as i sorta doubt it'll be anything that pops up immediately but on a technical level i think it should be feasible...
